awswrangler/s3/_read_parquet.py

```diff
@@ -237,8 +237,15 @@ def _read_parquet_chunked(
             chunks = pq_file.iter_batches(
                 batch_size=batch_size, columns=columns, use_threads=use_threads_flag, use_pandas_metadata=False
             )
+
+            if columns is not None:
```

Comment:
Minor but this can be just a one-liner that also preserves schema metadata, like in [_read.py:L236](https://github.com/aws/aws-sdk-pandas/blob/main/awswrangler/s3/_read.py#L236)
```
if columns:
    schema = pa.schema([schema.field(column) for column in columns], schema.metadata)
```